Sources/App/Core/PackageCollection+generate.swift

```diff
@@ -294,8 +294,10 @@ private extension Array where Element == PackageCollection.Compatibility {
             // Gather up builds via a Set to de-duplicate various
             // macOS build variants - spm, xcodebuild, ARM
             Set<Pair>(
-                builds.map { Pair.init(platform: .init(platform: $0.platform),
-                                       version: $0.swiftVersion.displayName) }
+                builds
+                    .filter { $0.status == .ok}
```

Comment:
Some lines are more important than others! 😬

Reply:
Aha, the smallest things 'eh

Thanks for getting this one fixed so quickly Sven!